ws-server/src/tests.rs

```diff
@@ -173,8 +173,6 @@ async fn can_set_max_connections() {
 	assert!(conn3.is_err());
 	let err = conn3.unwrap_err();
 	assert!(err.to_string().contains("WebSocketHandshake failed"));
-	assert!(err.to_string().contains("Connection reset by peer"));
-	// Err(Io(Os { code: 54, kind: ConnectionReset, message: \"Connection reset by peer\" }))");
```

Comment:
windows returns https://support.microsoft.com/en-us/topic/an-application-may-receive-the-10054-error-when-the-application-receives-data-from-a-connection-on-a-computer-that-is-running-windows-7-or-windows-server-2008-r2-if-a-tdi-filter-driver-is-installed-73fd74a9-51c2-3663-e965-e1187b1cfc92

so let's remove the assertion it would be better if we used `io::Error` in the test client I suppose